packages/http-urlencode-path-parser/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = () => ({
+  before: (handler, next) => {
+    if (handler.event.pathParameters) {
+      for (let key in handler.event.pathParameters) {
+        try {
+          handler.event.pathParameters[key] = decodeURIComponent(handler.event.pathParameters[key])
+        } catch (e) {
+          console.error(e)
```

Comment:
How about you introduce flag here, for example `shouldThrow` and check it here, or better yet, maybe errorHandler, like:
```
handler.use(urlEncodePathParser({
    onError: (event, context) => {
         // do something here
    }
}))
```
Just a suggestion, this is something that I would find useful if I were to use this plugin - otherwise you won't know if this function worked or not

Reply:
I opted to a throw an error, so that the error handler middleware can take care of it.